terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -0,0 +1,37 @@
+import graphene
+from graphene import relay
+from graphene_django import DjangoObjectType
+
+from apps.soilproj.models import Site
+
+from .commons import BaseWriteMutation, TerrasoConnection
+
+
+class SiteNode(DjangoObjectType):
+    id = graphene.ID(source="pk", required=True)
+
+    class Meta:
+        model = Site
+
+        filter_fields = {"name": ["icontains"]}
+        fields = ("name", "lat_deg", "lon_deg")
+
+        interfaces = (relay.Node,)
+        connection_class = TerrasoConnection
+
+
+class SiteAddMutation(BaseWriteMutation):
```

Comment:
Suggestion: I see some mutation inheriting from `BaseMutation` and others from `BaseWriteMutation` we should try to use `BaseWriteMutation` as much as possible